terra/contracts/wormhole/src/error.rs

```diff
@@ -67,6 +67,11 @@ pub enum ContractError {
     #[error("GuardianSetIndexIncreaseError")]
     GuardianSetIndexIncreaseError,
 
+
+    /// Guardian set must not be zero
+    #[error("GuardianSetZeroAddresses")]
```

Comment:
There's no harm in making the actual error message more detailed:

```
#[error("Guardian set must not be empty")]
GuardanSetZeroAddresses,
```

Reply:
fixed.